tests/navigation/test_navigation.py

```diff
@@ -0,0 +1,85 @@
+from direct.showbase.ShowBase import ShowBase
+from panda3d import navigation
+from panda3d import core
+from panda3d.core import NodePath
+
+def test_input_geom():
+	
+    data = core.GeomVertexData("", core.GeomVertexFormat.get_v3(), core.Geom.UH_static)
+    vertex = core.GeomVertexWriter(data, "vertex")
+    vertex.add_data3((0, 0, 0))
+    vertex.add_data3((100, 1000, 0))
+    vertex.add_data3((0, 100, 2))
+    vertex.add_data3((500, 200, 2))
+    vertex.add_data3((170, 20, 2))
+    vertex.add_data3((100, 100, 100))
+
+    prim = core.GeomTriangles(core.Geom.UH_static)
+    prim.add_vertices(0, 1, 4)
+    prim.closePrimitive()
+
+    geom = core.Geom(data)
+    geom.add_primitive(prim)
+
+    prim.add_vertices(2,3,5)
+    prim.closePrimitive()
+    geom.add_primitive(prim)
+
+    node = core.GeomNode('gnode')
+    node.addGeom(geom)
+
+    scene = NodePath(node)
+    # Defining navmesh as object of NavMehsBuilder class.
+    nav = navigation.NavMeshBuilder()
```

Comment:
I think we should avoid starting a convention where we call `NavMeshBuilder` instances `nav` since that is better suited for the nav mesh itself. Maybe `nav_builder` or just `builder`?